doc/setup/install.rst

```diff
@@ -530,7 +531,8 @@ on the requirements file:
 .. code-block:: console
 
     $ cd /CommunityPath
-    $ pip3 install wheel -r requirements.txt
+    $ pip3 install setuptools wheel
```

Comment:
I would prefer `apt install python3-setuptools python2-wheel`

Reply:
@d-fence But what if the user is using a virtualenv which sometimes does not recognize global packages properly ?
It seems cleaner to me to keep setuptools and wheel as Python dependencies since they're needed to install Python packages than to list them as Odoo's Debian dependencies. But that's just a matter of philosophy. The outcome remains the same.